### ISSUE TITLE
[FEAT] Different icons in dark mode

### ISSUE THREAD

**Issue by Davo00** [ThomasFKJorna/zotero-night#136]
**Is your feature request related to a problem? Please describe.**
Currently the icons used are defaults by zotero, this look very ugly and does not fit the theme.
**Describe the solution you'd like**
Use another icon set, which does fit to the dark night theme

**Comment by tefkah** [comment]
Hmm, interesting idea, im not opposed to it, but I worry that too much will break if I try to do this, like maybe some icons dissapearing.

The zotero devs already get mad at me bc sometimes my plugin breaks things, but instead of complaining to me they go to the zotero forums, which then requires the devs to check why it happens only to conclude it was this plugin.

I do like the idea though, I might experiment with it in a future release

**Comment by tefkah** [comment]
Maybe the icons from the Web version might work

![image](https://github.com/tefkah/zotero-night/assets/21983833/09f8dd5d-6af2-469a-9441-d5c6f21ec1a3)

**Comment by Davo00** [comment]
> Maybe the icons from the Web version might work
> 
> ![image](https://user-images.githubusercontent.com/21983833/263072881-09f8dd5d-6af2-469a-9441-d5c6f21ec1a3.png)

Yes, those are beautiful. Would it be possible to have them for the desktop version?
